### ISSUE TITLE
Steam install on multiple drive - Games shows as not installed

### ISSUE THREAD

**Issue by etshy** [JosefNemec/Playnite#1253]
It seems Playnite have some difficulties with multiple steam games directory.
I have some games that are installed on antoher drive than the "default" (the first used) and for playnite they are not installed

Example with two screens below

on Steam (installed and can play)
https://i.imgur.com/WjOjYRy.jpg

on Playnite (not installed)
https://i.imgur.com/N73ieGD.jpg

I can changed the game properties in Playnite to set it as installed and add the eexe path but It's not ideal to do that manually.

**Comment by JosefNemec** [comment]
Can you go to `About` menu and create diag. package for me please? This should be working just fine if the Steam installation is not broken in some way.

**Comment by etshy** [comment]
Her eis the diag ID : 8a596f76-cdd2-400b-95b8-1742b4ca4bb0

The game is launching fine when i start it via steam so it should be all fine, I guess.

**Comment by JosefNemec** [comment]
Thanks. Sadly I don't see anything wrong in logs. Can you to go Steams installation folder and check that `SteamApps\libraryfolders.vdf` file contains pats to your library on other drive? Also is Tropico the only game causing issues?

**Comment by etshy** [comment]
I had the problem on Total War Shogun2 too but I manually checked the installed state and add installation path and play action
I didn't have the probleme on Insurgency Sandstorm, Cossacks 3 and Total war Three kingdoms (cossacks 3 TWTK were very recently installed)

here is what my `libraryfolders.vdf` file contains
```
"LibraryFolders"
{
	"TimeNextStatsReport"		"1563999636"
	"ContentStatsID"		"-645612242716365328"
	"1"		"G:\\SteamLibrary"
}
```
Tropico 6 is installed on G: Disk
Steam is installed on D: Disk (I also have some games installed on D: disk, but the disk doesn't appear in this file.

I just checked Tropico 6 on Playnite and it appears as installed now.
Maybe it was just a library update that goes wrong and the recent one fixed the problem.
As I don't have the problem anymore I'm closing the issue.
